credentialdigger/cli/hook.py

```diff
@@ -0,0 +1,180 @@
+"""
+The 'hook' module can be used to run credential digger as a pre-commit hook.
+It detects hardcoded secrets in staged files.
+
+NOTE: It uses SQLite and the discoveries are saved (by default)
+in /home/USER/.local/data.db
+
+usage: credentialdigger hook [-h] [--dotenv DOTENV] [--rules RULES]
+                             [--db_path DB_PATH] [--no_interaction]
+
+optional arguments:
+  -h, --help         show this help message and exit
+  --dotenv DOTENV    The path to the .env file which will be used in all
+                     commands. If not specified, the one in the current
+                     directory will be used (if present).
+  --rules RULES      Specify the yaml file path containing the scan rules
+                     e.g., /path/to/rules.yaml
+  --db_path DB_PATH  Specify the database file path where to save the results
+                     e.g., /path/to/data.db
```

Comment:
can't we use `--sqlite` instead of `--db_path` to be consistent to the other CLIs? You did the same with `--dotenv`